db/migrate/20230127035924_remove_users_from_likes.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUsersFromLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, foreign_key: {to_table: :users}
+  end
+end
```

Comment:
- [ ] @Ngufor-emmanuella kindly check your migration files am getting the error below when running the migrations ,may be caused by having  2 migration files having same code as seen in screeenshot below
![image](https://user-images.githubusercontent.com/61408860/219141768-6d0c8fbe-d08a-446b-854a-24f4ae33ff55.png)
![image](https://user-images.githubusercontent.com/61408860/219141812-a7c010d2-c9eb-4787-a2b5-1020a016aa37.png)

![image](https://user-images.githubusercontent.com/61408860/219141615-8294a028-a745-4203-b6a0-b6f3550f05db.png)